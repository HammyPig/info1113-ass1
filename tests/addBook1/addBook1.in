ADD BOOK filename 111111
add book filename 111111
aDD bOOK Filename 111111