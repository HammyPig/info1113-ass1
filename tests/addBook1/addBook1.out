user: No such file.

user: No such file.

user: No such file.

user: Ending Library process.
